### DISCUSSION TITLE
Possible bug: unconsumed not working properly

### DISCUSSION THREAD

**Discussion by fcoury** [ghostty-org/ghostty#3788] [Feature Requests, Ideas] [answered]
I have the following lines on my config file:

```
# Keybindings
keybind = unconsumed:ctrl+h=goto_split:left
keybind = unconsumed:ctrl+l=goto_split:right
keybind = unconsumed:ctrl+j=goto_split:bottom
keybind = unconsumed:ctrl+k=goto_split:top
```

I also use the same keybindings for tmux and I expected the keys to be passed along to tmux. When I have those keybindings on my config file, it seems like tmux fails to receive them.

Here's a screen recording of what it does with and without those lines:

https://github.com/user-attachments/assets/53bf918c-9d7f-49c3-bd3d-d8c8d4b45948

And my Ghostty version:

<img width="412" alt="image" src="https://github.com/user-attachments/assets/73b3bc39-7e7a-429d-a6b4-03d0cf85fb38" />

**Comment by fcoury** [comment]
I would love to open a proper issue for this, can anyone confirm they can replicate it?

**Reply by codegangsta** [reply]
I'm experiencing the same thing, only without tmux and just vim

**Comment by mitchellh** [comment, marked as answer]
This has been resolved in later versions of the macOS app where unconsumed will no longer be captured by menu items.
